src/pretix/settings.py

```diff
@@ -525,12 +525,13 @@
     ('es', _('Spanish')),
     ('tr', _('Turkish')),
     ('gl', _('Galician')),
+    ('cz', _('Czech')),
```

Comment:
While the country code of the Czech Republic is CZ, I believe the ISO-assigned language code of the Czech language is cs ;) Not at all confusing, I know.

Reply:
fixed.